### PR TITLE
Yooan

### LINKED ISSUE #236: Help me please
* **Is your feature request related to a problem? Please describe.**

* **Describe the solution you'd like**

* **Describe alternatives you've considered**

* **Additional context**

1. open file
2. open r
3. do some [stuff]

[https://www.youtube.com/](url)

- [ ] Fix problem
- [ ]  check to have eaten
clown-jpg-1664513530

### PR DESCRIPTION
**Pull request by yooanean**
## Proposed changes

Added **comments** to certain files. 

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [x] Documentation Update (change to naming or other documentation)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR._

- [ ] I have added unit tests that prove my fix is effective or that my feature works
- [x] i have added some lines of comments to files
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have checked that my changes have not broken any other functionality
- [x] should have checked my changes hasn't broken any other functionality but haven't
- [ ] I have linked to any issues this PR fixes 

## Points for review

Add checkboxes here for any specific aspects of your PR you would like to be checked in peer review. If you don't specify anything here, the reviewer will check:

- [ ] All of your unit tests pass
- [ ] Your code is clear and easy to understand
- [ ] You have documented any new features
- [ ] They can run the code you have written
- [ ] Your changes do not break any other functionality

## Who is reviewing this PR?

Tag your reviewer here to ensure they get a notification!

@ManiScriptWizard

### CODE REVIEW

**Comment by yooanean** [comment]
check this:
https://github.com/department-for-transport/git-and-github-training/blob/29df67bcd6a87bf3650f2afbe44b492db75090e7/email_search.R#L79-L84

File: email_search.R [hunk 1]
```diff
@@ -78,3 +78,7 @@ save_attachments(search_string = "Absence Dashboard",
 save_attachments(search_string = "TfL COVID-19 update report", 
                  file_type = "docx", 
                  folder = "G:/AFP/IHACAll/IHAC/015 DDU/005 Covid reporting/0001 R Projects/transport_modes_table/Data/Face coverings/TfL")
+
+
```
Comment (ManiScriptWizard) [inline email_search.R:82]: no need for white space

File: report_functions.R [hunk 2]
```diff
@@ -226,3 +226,7 @@ extract_commentary <- function(data, mode){
     dplyr::filter(Mode == mode) %>% 
     dplyr::pull(Commentary)
 }
+
+
+#some more random comments
```
Comment (ManiScriptWizard) [inline report_functions.R:231]: looks good

**Review by ManiScriptWizard — APPROVED** [review]
good code

**Pull request closed without merging**